x/gov/src/types/deposit.rs

```diff
@@ -0,0 +1,23 @@
+use gears::types::{address::AccAddress, base::coin::Coin};
+use serde::{Deserialize, Serialize};
+
+use crate::keeper::KEY_DEPOSIT_PREFIX;
+
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
+pub struct Deposit {
+    pub proposal_id: u64,
+    pub depositor: AccAddress,
+    pub amount: Vec<Coin>,
+}
+
+impl Deposit {
+    pub(crate) fn key(&self) -> Vec<u8> {
+        [
+            KEY_DEPOSIT_PREFIX.as_slice(),
+            &self.proposal_id.to_be_bytes(),
+            &[self.depositor.len() as u8], // We save 'cause `AccAddress` len shoudn't be bigger than 255
```

Comment:
len() returns a u8, so we don't need the cast

Reply:
Unexpected for me...

Reply:
Removed